api/functions/update_user_role.py

```diff
@@ -2,6 +2,7 @@
 from db import db
 from models import Users as User
 from graphql import GraphQLError
+from user_roles import *
```

Comment:
Is it possible to be more specific with imports, sometimes when importing all you can overwrite changes that are required that occurred in other files